### PR TITLE
istio.md: correct a typo

### PR DESCRIPTION
**Pull request by yzgyyang**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by kimsauce** [comment]
@yzgyyang thanks for your contribution! Can you please [sign our CLA](https://docs.google.com/forms/d/e/1FAIpQLScRonzY0gLo7swLExL8CeLLNvdvEkRtIjJLUrFalo0vaAdp7g/viewform) so we can get your PR merged?

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/istio.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ To send data to Sumo Logic collector either in a Kubernetes or standalone setup,
 
 ### OpenCensus Agent exporter
 
-Set`meshConfig.enableTracing=true` and `meshConfig.defaultConfig.tracing.openCensusAgent.zipkin.address=` to the endpoint of the receiving collector:
+Set`meshConfig.enableTracing=true` and `meshConfig.defaultConfig.tracing.openCensusAgent.address=` to the endpoint of the receiving collector:
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/istio.md:20]:
```suggestion
Set `meshConfig.enableTracing=true` and `meshConfig.defaultConfig.tracing.openCensusAgent.address=` to the endpoint of the receiving collector:
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
CLA must be signed

**Comment by yzgyyang** [comment]
@kimsauce I've signed the CLA, thanks!

**Comment by kimsauce** [comment]
> @kimsauce I've signed the CLA, thanks!

thank you for the quick response! I will get this merged.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**